tests/tests_enabled.yml

```diff
@@ -25,6 +25,8 @@
     - name: Check /etc/rsyslog.d
       command: ls /etc/rsyslog.d
       changed_when: false
+      # rsyslog package may not be installed
+      failed_when: false
```

Comment:
would prefer not to use `failed_when: false`

Reply:
Heh, for Ansible I need to unlearn "don't do TOCTOU" :grin: I added a separate stat check instead.